src/toxcore/dht/server/mod.rs

```diff
@@ -384,23 +393,56 @@ impl Server {
         self.send_to(target_peer.saddr, nodes_req)
     }
 
-    /// send NatPingRequests to peers every 3 seconds
-    pub fn send_nat_ping_req(&self, peer: PackedNode, friend_pk: PublicKey) -> IoFuture<()> {
-        let mut peers_cache = self.peers_cache.write();
+    // send NatPingRequests to all of my friends.
```

Comment:
Please fix comment. You don't only send NatPingRequests but also you do HolePunching

Reply:
done